### PR TITLE
Added postCommerceCLICommand. Fixes #177

### LINKED ISSUE #177: The library needs a POST method for Commerce CLI commands
Commerce support is being added to the CLI. It will need a POST method to support Commerce CLI commands.

### PR DESCRIPTION
**Pull request by rwalter215**
Added postCommerceCLICommand

## Description

* added postCommerceCLICommand
* added unit tests
* updated documentation
## Related Issue

 #177 

## Motivation and Context

Adds Commerce functionality to cli

## How Has This Been Tested?

Unit tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test/commerce.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { codes } = require('../src/SDKErrors')
+
+/* global createSdkClient */ // for linter
+
+test('postCLICommand - success', async () => {
```
Comment (thoughtassassin) [inline test/commerce.test.js:16]: Let's change the name of the test to be consistent with the method (e.g. postCommerceCLICommand)

**Review by thoughtassassin — CHANGES_REQUESTED** [review]

File: test/commerce.test.js (hunk 1 above)
Comment (rwalter215) [reply test/commerce.test.js]: Good catch, fixed.

File: src/index.js [hunk 2]
```diff
@@ -1225,6 +1225,23 @@ class CloudManagerAPI {
...
+  /**
+   * Make a Post to Commerce API
+   *
+   * @param {string} programId - the program id
+   * @param {string} environmentId - the environment id
+   * @param {object} requestBody - the request body
+   * @returns {Promise<object>} a truthy value
+   */
+  postCommerceCLICommand (programId, environmentId, requestBody) {
+    const resourcePath = `/program/${programId}/environment/${environmentId}/runtime/commerce/cli/`
```
Comment (justinedelson) [inline src/index.js:1238]: @rwalter215 this path should be derived from a HAL link as explained in https://github.com/adobe/aio-lib-cloudmanager/blob/main/.github/CONTRIBUTING.md#coding-patterns-and-styles

File: src/SDKErrors.js [hunk 3]
```diff
@@ -104,3 +104,4 @@ E('ERROR_DELETE_IP_ALLOWLIST_BINDING', 'Could not remove IP Allow List binding:
 E('ERROR_UNSUPPORTED_ADVANCE_STEP', 'Advancing the step %s is not supported in the CLI at present.')
 E('ERROR_REFRESH_STEP_STATE', 'Cannot refresh step state: %s')
 E('ERROR_STEP_STATE_NOT_RUNNING', 'The %s step in execution %s is not currently running.')
+E('ERROR_POST_COMMERCE', 'Could not post to Commerce endpoint %s')
```
Comment (justinedelson) [inline src/SDKErrors.js:107]: Since this error is specific to CLI commands, please use an enum name and value that reference CLI commands.

File: src/index.js (hunk 2 above)
Comment (rwalter215) [reply src/index.js:1238]: As of now we cannot get the path from a HAL link, that functionality has not been added to the api yet. Can I keep this code the way it is, or is this there a different approach you'd prefer.

Comment (justinedelson) [reply src/index.js:1238]: I can't think of another alternative.

Comment (rwalter215) [reply src/index.js]: I implemented retrieving the url from the HAL link, we'll just deal with this functionality not being available until the api supports it.

File: src/SDKErrors.js (hunk 3 above)
Comment (rwalter215) [reply src/SDKErrors.js]: Made requested changes to the enum name and value

**Comment by rwalter215** [comment]
Made requested changes, ready for re-review

File: test/jest/jest.fetch.setup.js [hunk 4]
```diff
@@ -1437,57 +1443,57 @@ beforeEach(() => {
     executionForPipeline7 = id
   }
   fetchMock.mock((url, opts) => url === 'https://cloudmanager.adobe.io/api/program/5/pipeline/7/execution/1005/phase/4596/step/8492/cancel' &&
-      opts.method === 'PUT' && opts.body === JSON.stringify({ cancel: true }),
+    opts.method === 'PUT' && opts.body === JSON.stringify({ cancel: true }),
```
Comment (justinedelson) [inline test/jest/jest.fetch.setup.js:1446]: Please revert these unrelated changes.

**Comment by justinedelson** [comment]
@rwalter215 Please just revert the unrelated changes to test/jest/jest.fetch.setup.js and I'll merge this.

**Comment by rwalter215** [comment]
Sounds good, thanks. I reverted unrelated changes in test/jest/jest.fetch.setup.js
